external-crates/move/crates/move-analyzer/prettier-move/tests/formatting/formatting.move

```diff
@@ -0,0 +1,8 @@
+// This is a line comment
```

Comment:
Please add a specific test for different types of comment embedding, for example, comments for function call arguments, comments interspersed with if statement conditions (and, more generally) inside other statements. Examples (not an exhaustive list):
```
foo(/* some comment */ true);

if (some_condition ||
    /* some comment */
    another_condition) {}

let m = /* some comment */ = 42;
```